os/include/arpa/inet.h

```diff
@@ -162,6 +162,9 @@ extern "C"
 /**
  * @brief Convert the unsigned integer netlong from network byte order to host byte order
  *
+ * @details @b #include <arpa/inet.h>\n
+ * SYSTEM CALL API\n
```

Comment:
please check if this API is SYSTEM CALL or not.

Reply:
revised